src/KeyVault/KeyVault/Commands/Secret/RemoveAzureKeyVaultSecret.cs

```diff
@@ -96,7 +108,16 @@ public override void ExecuteCmdlet()
                 Name = InputObject.Name;
             }
 
-            if(InRemovedState.IsPresent)
+            if (ParameterSetName == BySecretUriParameterSet)
```

Comment:
Looks there are same logics for secret related commands, please consider how to put these codes in a place. Like a method called PreprocessParameters() in base class

Reply:
@notyashhh, could you resolve this comment?